src/index.js

```diff
@@ -247,6 +249,9 @@ class Masq {
         if (!json.userAppDEK) {
```

Comment:
weh have many check like this, we can reduce LOCs by iterating on an array of properties to validate